roles/test-beat/templates/heartbeat.yml.j2

```diff
@@ -4,6 +4,8 @@ heartbeat.monitors:
 - type: http
   schedule: '@every 1s'
   urls: ["http://localhost:12345"]
+  fields:
```

Comment:
Do we support `fields` inside a monitor?

Reply:
I will put all of them at the global level.